src/Language/Haskell/Brittany/Internal/Layouters/Stmt.hs

```diff
@@ -26,51 +27,58 @@ import {-# SOURCE #-} Language.Haskell.Brittany.Internal.Layouters.Expr
 
 
 layoutStmt :: ToBriDoc' (StmtLR RdrName RdrName (LHsExpr RdrName))
-layoutStmt lstmt@(L _ stmt) = docWrapNode lstmt $ case stmt of
-  LastStmt body False _    -> do
-    layoutExpr body
-  BindStmt lPat expr _ _ _ -> do
-    patDoc <- fmap return $ colsWrapPat =<< layoutPat lPat
-    expDoc <- docSharedWrapper layoutExpr expr
-    docAlt
-      [ docCols
-        ColBindStmt
-        [ appSep patDoc
-        , docSeq [appSep $ docLit $ Text.pack "<-", docForceParSpacing expDoc]
+layoutStmt lstmt@(L _ stmt) = do
+  indentPolicy <- mAsk
+    <&> _conf_layout
+    .>  _lconfig_indentPolicy
+    .>  confUnpack
+  docWrapNode lstmt $ case stmt of
+    LastStmt body False _    -> do
+      layoutExpr body
+    BindStmt lPat expr _ _ _ -> do
+      patDoc <- fmap return $ colsWrapPat =<< layoutPat lPat
+      expDoc <- docSharedWrapper layoutExpr expr
+      docAlt
+        [ docCols
+          ColBindStmt
+          [ appSep patDoc
+          , docSeq [appSep $ docLit $ Text.pack "<-", docForceParSpacing expDoc]
+          ]
+        , docCols
+          ColBindStmt
+          [ appSep patDoc
+          , docAddBaseY BrIndentRegular
+            $ docPar (docLit $ Text.pack "<-") (expDoc)
+          ]
         ]
-      , docCols
-        ColBindStmt
-        [ appSep patDoc
+    LetStmt binds            -> layoutLocalBinds binds >>= \case
+      Nothing        -> docLit $ Text.pack "let" -- i just tested
+                                -- it, and it is
+                                -- indeed allowed.
+                                -- heh.
+      Just []        -> docLit $ Text.pack "let" -- this probably never happens
+      Just [bindDoc] -> docAlt
+        [ docCols
+          ColDoLet
+          [ appSep $ docLit $ Text.pack "let"
+          , docSetBaseAndIndent $ return bindDoc
```

Comment:
`docSetBaseAndIndent` there most likely produces ~~hanging indentation~~ (or at least) more-than-usual indentation. Probably need to disable this alt-branch, too.